scripts/installation/install-openkat.yml

```diff
@@ -0,0 +1,588 @@
+# Ansible playbook to install OpenKAT
+#
+# Copyright (C) 2023  Jeroen Baten <jbaten@i2rs.nl>
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301, USA
+#
+# Changelog
+# Date       Author   Comments
+# ===========================================================================
+# 2023-08-12 J. Baten Initial version
+# 2023-08-14 J. Baten Debugging. Hacking Superadmin password to support 
+#                     current stable standard Ansible
+# 2023-09-16 J. Baten Inject variables for version numers, and prepare for MR.                    
```

Comment:
Changes are already tracked by git, duplicating this information in the file isn't really useful.